common/script/src/content/quests/gold-purchasable.js

```diff
@@ -0,0 +1,119 @@
+import {each, defaults} from 'lodash';
+import {
+  translator as t,
+  merge,
+  formatForTranslator
+} from '../helpers';
+
+let dilatoryDistressSeries = {
```

Comment:
If we make quest series into arrays, we could handle the sequence via index rather than the weird `previous` attribute.